modules/imap/site.js

```diff
@@ -1301,31 +1301,90 @@ var imap_screen_email = function() {
 
 var add_email_in_contact_trusted = function(list_email) {
     if (list_email) {
-      Hm_Ajax.request(
-        [
-          { name: 'hm_ajax_hook', value: 'ajax_add_contact' },
-          { name: 'email_address', value: list_email.join(',') },
-        ],
-        function (res) {
-          window.location.reload();
-        }
-      );
+        Hm_Ajax.request(
```

Comment:
What did you change here, seams to be indentation. make sure you remove changes like this.

Reply:
okay

Reply:
@christer77 Please check this image to see what I’m seeing on my side. No changes were made, except for a space you added before `Hm_Ajax.request(` on the line following `if (list_email) {`.
<img width="920" alt="Screenshot 2024-11-28 at 21 26 15" src="https://github.com/user-attachments/assets/cbf07c02-a0ce-4d57-973f-18df0b885495">